### DISCUSSION TITLE
Appending CSVs ? [Apr 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1686] [Discord Threads]
> 🙋‍♂️ **vikiiingur**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### did you mean to just append the data in csv or update the existing records as well ?  
#  
> 💁 **o1lab**  
#### please note the upload csv part is in alpha yet 🙂  
#  
> 🙋‍♂️ **vikiiingur**  
#### yes, append is the correct word for what I want to achieve  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/961627614034944131)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
